cmd/run.go

```diff
@@ -63,6 +64,7 @@ const (
 	invalidConfigErrorCode       = 104
 	externalAbortErrorCode       = 105
 	cannotStartRESTAPIErrorCode  = 106
+	scriptExceptionErrorCode     = 107
```

Comment:
This should probably be 108, since we already use 107 in https://github.com/k6io/k6/pull/1920 (we probably should also move these to the `consts` package eventually)

Reply:
I don't think it really matters who gets `107` and who gets `108`. 

Whoever is merged second will have a conflict and will need to resolve it.

Given that IMO this is more likely to be merged now I prefer to no add a "hole" in the error codes ;)

Reply:
fair enough :+1: